test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -3531,6 +3531,13 @@ items:
     - ""
     attributeRestrictions: null
     resources:
+    - namespaces
```

Comment:
This gets added to drive the SCC admission plugin that the build controller embeds.